### DISCUSSION TITLE
How to scheduler DAG frequenct at odd minutes

### DISCUSSION THREAD

**Discussion by nitra-1** [apache/airflow#18198] [General] [answered]
I want to schedule the frequency to 5 minutes and want it to execute not exactly at multiples of 5, like i want to execute at for example 1.08 am then 1.13am ,1.18am so on.
Is this possible ? and how ?
Regards
Nitin

**Comment by turbaszek** [comment, marked as answer]
Use cron schedule of:
```
3/5 * * * *
```
This will run the DAG at:
```
2021-09-13 09:28:00
then at 2021-09-13 09:33:00
then at 2021-09-13 09:38:00
then at 2021-09-13 09:43:00
then at 2021-09-13 09:48:00
```

Every-time I need cron expression I use this awesome [crontab guru](https://crontab.guru/#3/5_*_*_*_*).

**Reply by nitra-1** [reply]
Thank you!! this works.
